Makefile

```diff
@@ -8,6 +8,8 @@ FIRST_GOPATH := $(firstword $(subst :, ,$(shell go env GOPATH)))
 OS ?= $(shell uname -s | tr '[A-Z]' '[a-z]')
 ARCH ?= $(shell uname -m)
 GOARCH ?= $(shell go env GOARCH)
+GOFLAGS := -tags=fips_enabled
+GOEXPERIMENT := boringcrypto
```

Comment:
Afaiu, this is a cgo impl, with some overhead. Which might be fine for token-refresher use cases.

Reply:
It doesn't appear to require cgo - the build target specifies CGO_ENABLED=0 and the output is a static binary that includes the boringcrypto symbols